src/redist/targets/GenerateBundledVersions.targets

```diff
@@ -119,7 +119,7 @@
 
       <AspNetCore31RuntimePackRids Include="@(AspNetCore30RuntimePackRids)" />
       <AspNetCore50RuntimePackRids Include="@(AspNetCore31RuntimePackRids);linux-musl-arm;win-arm64" />
-      <AspNetCoreRuntimePackRids Include="@(AspNetCore50RuntimePackRids)" />
+      <AspNetCoreRuntimePackRids Include="@(AspNetCore50RuntimePackRids)" /><!-- Add ;osx-arm64 when bits are available -->
```

Comment:
is this just till osx-arm64 asp.net packages propagate to sdk? They are already appearing in the .net 6 package source from what I can see.

Reply:
Yeah this too was temporary. We are still blocked on #8966 to get the aspnetcore osx-arm64 bits.